twitter_scraper/twitterscraper/insert_seed.py

```diff
@@ -0,0 +1,39 @@
+import json
```

Comment:
FYI: 
This is essentially the same as `cli` in go

Reply:
yes. it is kind of a leftover from the initial `twitterscraper`, when I developed it outside of the repo.
Should we remove it and add the `cli` to `docker-compose` ?